Stylesheet/css/general.css

```diff
@@ -2,7 +2,7 @@
 html, body {
     margin: 0;
     padding: 0;
-    font-size: 62.5%;
+    font-size: 100%;
 }
 
 .img-responsive {
```

Comment:
width: 100% makes the image take up the extra space it requires for it to maintain its aspect ration. use max-width: 100% to achieve responsiveness

[read this article for detailed explaination](https://forum.freecodecamp.org/t/what-is-the-difference-between-width-100-and-max-width-100-on-image-element-in-css/409205)